test/build.bats

```diff
@@ -186,7 +186,10 @@ OUT
   mkdir -p "$brew_libdir"
 
   stub uname '-s : echo Linux'
-  stub brew "--prefix libyaml : echo '$brew_libdir'" false
+  stub brew \
+    "--prefix libyaml : echo '$brew_libdir'" \
+    false \
+    "--prefix libyaml : echo '$brew_libdir'"
```

Comment:
1. `needs_yaml` call from the definition file
2. readline check from code
3. libyaml check from code